src/core/linalg/src/sparse/4C_linalg_fevector.hpp

```diff
@@ -0,0 +1,323 @@
+    {
+      return vector_->Import(A, Exporter, CombineMode, Indexor);
+    }
+
+    int export_to(const Epetra_SrcDistObject& A, const Core::LinAlg::Import& Importer,
+        Epetra_CombineMode CombineMode, const Epetra_OffsetIndex* Indexor = nullptr)
+    {
+      return vector_->Export(A, Importer.get_epetra_import(), CombineMode, Indexor);
+    }
+
+    int export_to(const Epetra_SrcDistObject& A, const Epetra_Export& Exporter,
+        Epetra_CombineMode CombineMode, const Epetra_OffsetIndex* Indexor = nullptr)
+    {
+      return vector_->Export(A, Exporter, CombineMode, Indexor);
+    }
+
+    int sum_into_global_value(int GlobalRow, int FEVectorIndex, double ScalarValue)
+    {
+      return vector_->SumIntoGlobalValue(GlobalRow, FEVectorIndex, ScalarValue);
+    }
+
+    int global_assemble(Epetra_CombineMode mode = Add, bool reuse_map_and_exporter = false)
+    {
+      return vector_->GlobalAssemble(mode, reuse_map_and_exporter);
+    };
+
+    int sum_into_global_value(long long GlobalRow, int FEVectorIndex, double ScalarValue)
+    {
+      return vector_->SumIntoGlobalValue(GlobalRow, FEVectorIndex, ScalarValue);
+    }
+
+    int sum_into_global_values(int numIDs, const int* GIDs, const int* numValuesPerID,
+        const double* values, int vectorIndex = 0)
+    {
+      return vector_->SumIntoGlobalValues(numIDs, GIDs, numValuesPerID, values, vectorIndex);
+    }
+
+    int sum_into_global_values(
+        int numIDs, const int* GIDs, const double* values, int vectorIndex = 0)
+    {
+      return vector_->SumIntoGlobalValues(numIDs, GIDs, values, vectorIndex);
+    }
+
+
+    int reciprocal_multiply(double ScalarAB, const Epetra_MultiVector& A,
+        const Epetra_MultiVector& B, double ScalarThis)
+    {
+      return vector_->ReciprocalMultiply(ScalarAB, A, B, ScalarThis);
+    }
+
+    int sum_into_local_value(int MyRow, int FEVectorIndex, double ScalarValue)
+    {
+      return vector_->SumIntoMyValue(MyRow, FEVectorIndex, ScalarValue);
+    }
+
+
+    int sum_into_local_value(
+        int MyBlockRow, int BlockRowOffset, int FEVectorIndex, double ScalarValue)
+    {
+      return vector_->SumIntoMyValue(MyBlockRow, BlockRowOffset, FEVectorIndex, ScalarValue);
+    }
+    /**
+     * View a given Epetra_FEVector object under our own FEVector wrapper.
+     */
+    [[nodiscard]] static std::unique_ptr<FEVector<T>> create_view(Epetra_FEVector& view);
+
+    [[nodiscard]] static std::unique_ptr<const FEVector<T>> create_view(
+        const Epetra_FEVector& view);
+
+
+   private:
+    FEVector() = default;
+
+    //! The actual Epetra_FEVector object.
+    Utils::OwnerOrView<Epetra_FEVector> vector_;
+
+    //! Map from Epetra_FEVector
+    mutable View<const Map> map_;
+
+    //! MultiVector view of the FEVector. This is used to allow implicit conversion to MultiVector.
+    mutable View<MultiVector<T>> multi_vector_view_;
+
+    friend class MultiVector<T>;
+  };
+
```

Comment:
```suggestion

  template <>
  struct EnableViewFor<Epetra_FEVector>
  {
    using type = FEVector<double>;
  };
```

Something like this is missing to enable the `View<>` mechanism.

Reply:
I think now all test should pass again.